src/lightning/pytorch/serve/servable_module.py

```diff
@@ -56,9 +57,11 @@ def configure_response(self):
         assert serve_cb.resp.json() == {"output": [0, 1]}
     """
 
+    @abstractmethod
```

Comment:
shouldn't this be done in a different pr? Seems unrelated to typing

Reply:
Mypy was complaining that these methods annotate that they return a Dict when the true return is None (empty body)